types/react-visibility-sensor/index.d.ts

```diff
@@ -1,5 +1,5 @@
 // Type definitions for react-visibility-sensor 3.11
-// Project: https://github.com/joshwnj/react-visibility-sensor
+// Project: https://github.com/joshwnj/react-visibility-sensor#readme
```

Comment:
Why did you add `#readme` to this?

Reply:
I first made the project based on an existing project. Then when I was done and about to submit I saw the check-box,

`[x ] Create it with dts-gen --dt, not by basing it on an existing project.`

I did that and diffed it with my version. The only difference was how the link in the header looked and the ordering of arguments in `tsconfig.json`. In the generated files it had the `#readme` at the end of the link, so I added it to conform with that template.